ios/browser/api/query_filter/query_filter_utils.mm

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/ios/browser/api/query_filter/query_filter_utils.h"
+#include "base/strings/sys_string_conversions.h"
```

Comment:
blank line between header and the rest of the includes

Reply:
Done